scripts/prepare_motus_crates/motus_ro_crates_preparer.py

```diff
@@ -5,13 +5,15 @@
 import logging
 import os
 from urllib.parse import urljoin
+from uuid import uuid4
 
 import requests
 import shutil
 import tarfile
 from ro_crate_ui_assets_provider import RoCrateUIAssetsProvider
 from tqdm import tqdm
 from bs4 import BeautifulSoup
+from arcp import *
```

Comment:
A bit of a pedantic comment, I think it's better to do `from arcp import arcp_location` (which I think it's the only element from the arcp this code is using)

Reply:
I've optimised this import as suggested.

Thanks